docs/modules/clusters/pages/ucn-dynamic-config.adoc

```diff
@@ -39,7 +39,7 @@ public class Client {
 <1> Fetches the dynamic {ucn} configuration instance from the member
 <2> Creates a new `UserCodeNamespaceConfig` with the name `my_namespace`
 <3> Adds a `Class` resource to the `my_namespace` config, where the `Class` resolved from the client's local context
-<4> Adds a `Class` resource using the class identifier
+<4> Adds a `Class` resource using the URL to the _.class_ and the supplied identifier. In this case, a null identifier is valid; if used, the ID is used as the class name
 <5> Adds the `my_namespace` configuration with its resources to the parent, which is dynamically applied on the member
```

Comment:
Nit-pick - wording is slightly confusing - 

 "; if null is supplied, the class name is used as the ID"
 
 wdyt?